aiida/daemon/execmanager.py

```diff
@@ -806,6 +771,7 @@ def retrieve_computed_for_authinfo(authinfo):
                         calc.pk, retrieved_files.dbnode.pk),
                                      extra=logger_extra)
                     retrieved_files.store()
+
```

Comment:
Add a comment that we are not storing the _temp files

Reply:
Thanks, fine for me. I'll approve this and let you merge or decide to put an explicit clear of the SandboxFolder for the temporary FolderData